installers/kubectl/client-setup.sh

```diff
@@ -12,41 +12,49 @@
 #  limitations under the License.
 
 # This script should be run after the operator has been deployed
-export PGO_OPERATOR_NAMESPACE=${PGO_OPERATOR_NAMESPACE:-pgo}
+export PGO_OPERATOR_NAMESPACE="${PGO_OPERATOR_NAMESPACE:-pgo}"
+PGO_USER_ADMIN="${PGO_USER_ADMIN:-pgouser-admin}"
```

Comment:
should there be an export here?

Reply:
or alternatively remove the export on `PGO_OPERATOR_NAMESPACE`

Reply:
`PGO_USER_ADMIN` does not need to be exported because it's never used anywhere else but here. Perhaps it should be lowercased, but I leave that to the better Bashers than me.

Looking through the code, we don't need to export `PGO_OPERATOR_NAMESPACE` unless one is running the tests. I would suggest we leave it given it doesn't hurt.

Reply:
Is it the scripts job to setup a users environment? If so, should we also automatically export the variables at the bottom?

Reply:
Yeah...I don't think `PGO_OPERATOR_NAMESPACE` is used unles you're testing (which is a developer task). That's why I think it doesn' thurt to have it available. Should we put it into the environment in general? 🤔 tough one. Will think on it.